modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaSetResultsAndTotalCheck.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.CharPool;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.source.formatter.check.util.JavaSourceUtil;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaSetResultsAndTotalCheck extends BaseFileCheck {
+
+	@Override
+	protected String doProcess(
+			String fileName, String absolutePath, String content)
+		throws Exception {
+
+		if ((content == null) || !fileName.endsWith(".java")) {
+			return content;
+		}
+
+		content = _replaceSetResults(content);
+		content = _removeSetTotal(content);
+
+		return content;
+	}
+
+	private String _removeSetTotal(String content) {
+		String newContent = content;
+		Matcher setTotalMatcher = _setTotalPattern.matcher(content);
+
+		while (setTotalMatcher.find()) {
+			String methodCall =
+				JavaSourceUtil.getMethodCall(content, setTotalMatcher.start()) +
+					StringPool.SEMICOLON + StringPool.NEW_LINE;
+
+			if (_validateClassName(methodCall, "SearchContainer", content)) {
+				newContent = StringUtil.removeSubstring(
+					content, "\t\t" + methodCall);
```

Comment:
@TamyrisTorres, do we really need the `\t\t`? Not necessarily it will always exist in the customer's code, right?